Generator/ProductGenerator.php

```diff
@@ -86,12 +89,18 @@ class ProductGenerator implements GeneratorInterface
     /** @var CategoryRepositoryInterface */
     protected $categoryRepository;
 
+    /** @var GroupRepositoryInterface */
+    protected $groupRepository;
+
     /** @var Faker\Generator */
     protected $faker;
 
     /** @var array */
     protected $categoryCodes;
 
+    /** @var VariantGroupDataProvider[] */
+    protected $variantGroups;
```

Comment:
weird variable name for an array of providers i would expect an array of variant groups when reading it, please change